ui/components/app/flask/experimental-area/index.scss

```
@@ -0,0 +1,44 @@
+.experimental-area {
+  color: $flask-purple;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  .logo {
+    padding: 20px;
```

As a general rule we should be using pixel values that can be divisible by 8 for padding and margin. So it adheres to the 8px grid. Guidelines are a little lacking on this but [here is a link](https://www.notion.so/Extension-Frontend-Engineering-Guide-b82ddb3e14004b5db3799a9b446294a9#751df942ee8644bcb1af701650ebee9e)

```suggestion
    padding: 16px;
```